arm-resources/resources/2016-09-01/swagger/resources.json

```diff
@@ -1853,6 +1853,7 @@
           }
         },
         "debugSetting": {
+          "x-ms-client-flatten": true,
```

Comment:
I'm fine with removing the extension and can update this PR if that is the recommendation.